projects/Mallard/src/components/article/html/components/images.ts

```diff
@@ -198,7 +198,8 @@ const ImageBase = ({
         <figure class="image" data-role="${role || 'inline'}">
             <img
                 src="${path}"
-                onclick="window.ReactNativeWebView.postMessage(JSON.stringify({type: 'openLightbox', index: ${index}, isMainImage: 'false'}))"
+                onclick="window.ReactNativeWebView.postMessage(JSON.stringify({type: 'openLightbox', index: ${index ||
```

Comment:
It looks like`index` also being used in a line below. Could that also affect there when `index` is undefined? Would be good to know why it is undefined for cartoon.

Reply:
Good point, index is optional and was added for lightbox
Instead of doing this I have given the cartoon image an index